en/docs/reference/mediators/script-mediator.md

```diff
@@ -97,11 +91,14 @@ JavaScript, Groovy, or Ruby.
     </tr>
     <tr class="odd">
     <td>JavaScript</td>
-    <td>The JavaScript/E4X support is enabled by default in the Micro Integrator and ready for use.</td>
+    <td>The JavaScript based on GraalVM JS is enabled by default in the Micro Integrator and ready for use.</td>
     </tr>
     </tbody>
     </table>
 
+    !!! Note
+         The language `nashornJs` is deprecated from the JDK in Java 15. If you are using JDK 15 or above, you need to manually copy the [nashorn-core](https://mvnrepository.com/artifact/org.openjdk.nashorn/nashorn-core/15.4) and [asm-util](https://mvnrepository.com/artifact/org.ow2.asm/asm-util/9.5) jars to the <code>&lt;MI_HOME&gt;/lib</code> directory since Nashorn was [removed](https://openjdk.org/jeps/372) from the JDK.
```

Comment:
```suggestion
         The `nashornJs` language is deprecated in JDK 15 and above. If you are using JDK 15 or above, you need to manually copy the [nashorn-core](https://mvnrepository.com/artifact/org.openjdk.nashorn/nashorn-core/15.4) and [asm-util](https://mvnrepository.com/artifact/org.ow2.asm/asm-util/9.5) JAR files to the <code>&lt;MI_HOME&gt;/lib</code> directory since Nashorn was [removed](https://openjdk.org/jeps/372) from the JDK.
```